packages/common-library/src/common_library/user_messages.py

```diff
@@ -0,0 +1,11 @@
+def user_message(msg: str, *, _version=None) -> str:
+    """Marks a message as user-facing
+
+    Arguments:
+        msg -- human-friendly string that follows docs/user-messages-guidelines.md
+        _version -- version number to track changes to messages; increment when modifying an existing message
```

Comment:
annotation typing missing. And probably you need to precise that it cannot be negative ;)
And the default of None, means there is no version.